lab_1_keywords_tfidf/main.py

```diff
@@ -82,24 +134,49 @@ def calculate_tf(frequencies: dict[str, int]) -> Optional[dict[str, float]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if isinstance(frequencies, dict):
```

Comment:
revert the condition and return None immediately

Reply:
done